Neos.Neos/Classes/Fusion/Cache/GraphProjectorCatchUpHookForCacheFlushing.php

```diff
@@ -264,7 +264,7 @@ public function onAfterCatchUp(): void
     {
         foreach ($this->flushNodeAggregateRequestsOnAfterCatchUp as $request) {
             // We do not need to flush single node aggregates if we flush the whole workspace anyway.
-            if (!$this->hasMatchingFlushWorkspaceRequest($request)) {
+            if (!isset($this->flushWorkspaceRequestsOnAfterCatchUp[$request->workspaceName->value])) {
```

Comment:
Well, I'm not sure if this is more describing than before. 🤷‍♂️

Reply:
i think so too, otherwise its rather a "shallow" method (see A Philosophy of Software Design) and at least i had to first read the method to understand what it does so the method itself became useless. But thats just my take. Sorry feel free to overrule me and ill drop my commit :D